src/test/components/NetworkChart.test.js

```diff
@@ -222,140 +230,126 @@ describe('NetworkChartRowBar URL split', function() {
         endTime: 90,
       }
     );
-    expect(
-      // Find the URL shortening parts
-      networkChart
-        .find('.networkChartRowItemLabel span span')
-        .map(node => [node.prop('className'), node.text()])
-    ).toEqual([]);
+    expect(getUrlShorteningParts()).toEqual([]);
   });
 });
 
 describe('NetworkChartRowBar MIME-type filter', function() {
   it('searches for img MIME-Type', function() {
-    const { networkChart } = setupWithPayload(
-      'Load 101: htps://test.mozilla.org/img/optimized/test.png',
+    const { container } = setupWithPayload(
+      'Load 101: https://test.mozilla.org/img/optimized/test.png',
       {
         type: 'Network',
-        URI: 'https://mozilla.org/img/test123.png',
-        RedirectURI: 'https://mozilla.org/img/optimized',
+        URI: 'https://test.mozilla.org/img/optimized/test.png',
```

Comment:
This wasn't strictly necessary, but I decided to match the `URI` with the title above.

Because the test was working before, this means that our code is matching the mime type against the label and not the URI, and I'm a bit surprised about that.

Also I converted these markers to not be a redirection.